next/schema-generator/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  trailingComma: 'all',
+  tabWidth: 2,
+  semi: false,
+  singleQuote: true,
+  printWidth: 100,
+}
```

Comment:
I think you can put this top level and it will work well for both next and this directory ?

Reply:
Related to "robust solution".